src/tools/CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ IF (CARES_BUILD_TOOLS)
 	)
 	TARGET_COMPILE_DEFINITIONS (ahost PRIVATE HAVE_CONFIG_H=1)
 	TARGET_LINK_LIBRARIES (ahost PRIVATE ${PROJECT_NAME})
+	IF (CARES_STATIC)
```

Comment:
This does not look right to me. This flag being enabled does not imply there's only a static build, but that a static library is _also_ being built. So this change unconditionally adds these definitions and these linkages if there happens to be a static library built together with the shared one?